iris-GmbH/meta-iris-base
# Feature/tno/rdphoen 1289 dropbear host key pr

**Avoid the warning: REMOTE HOST IDENTIFICATION HAS CHANGED on development images:**

```
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the RSA key sent by the remote host is
```

CI: https://gitlab.devops.defra01.iris-sensing.net/MIRROR/iris-kas/-/pipelines/1996

**Test**
1. Flash-Boot with maintenance:
   - Key is generated and stored on first boot in /mnt/iris/dropbear
   - Make sure the key in /mnt/iris/dropbear is used
2. Flash-Boot with deploy: No dropbear at all
3. NFS-Boot with maintenance -> pre generated key from /etc/dropbear is used (/mnt/iris is not mounted)
4. Make sure there is not pre generated key on R1 images

**Helper**
Stop dropbear: `/etc/init.d/dropbear stop`
Start dropbear and create key if not present: `/etc/init.d/dropbear start`
